src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/JsonConverter.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace System.Text.Json.Serialization
+{
+    public static partial class JsonConverter
```

Comment:
Name this file `JsonConverter.Helpers.cs` given it looks like it contains private static methods, or  `JsonConverter.Common.cs`